### ISSUE TITLE
Failed to activate the view-tail-large-files package

### ISSUE THREAD

**Issue by hstove** [mark-hahn/view-tail-large-files#14]
After installing the latest update:

**Atom Version**: 0.187.0
**System**: Mac OS X 10.10.2
**Thrown From**: [view-tail-large-files](https://github.com/mark-hahn/view-tail-large-files) package, v0.1.13

### Stack Trace

Failed to activate the view-tail-large-files package

```
At Cannot read property 'message' of undefined

TypeError: Cannot read property 'message' of undefined
  at PluginMgr.activate (/Users/hank/.atom/packages/view-tail-large-files/lib/plugin-mgr.coffee:33:79)
  at ViewTailLargeFiles.activate (/Users/hank/.atom/packages/view-tail-large-files/lib/view-tail-large-files.coffee:20:16)
  at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app/src/package.js:240:19)
  at /Applications/Atom.app/Contents/Resources/app/src/package.js:221:30
  at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app/src/package.js:167:15)
  at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app/src/package.js:214:14)
  at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/src/package-manager.js:465:21)
  at /Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:226:29
  at exit (/Applications/Atom.app/Contents/Resources/app/node_modules/settings-view/lib/package-manager.js:43:16)
  at BufferedProcess.triggerExitCallback (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:63:47)
  at /Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:70:18
  at Socket.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/buffered-process.js:150:18)
  at Socket.emit (events.js:138:20)
  at Pipe.close (net.js:477:12)

```

### Commands

```
     -0:30.9 settings-view:open (atom-text-editor.editor)
     -0:29.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-base16-tomorrow-dark-theme.theme-atom-dark-ui)
     -0:06.0 editor:delete-to-beginning-of-line (atom-text-editor.editor.mini)
     -0:03.1 core:backspace (atom-text-editor.editor.mini)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "base16-tomorrow-dark-theme"
    ],
    "disabledPackages": [
      "emmet"
    ]
  }
}
```

### Installed Packages

```coffee
# User
RubyMotion, v0.6.2
Stylus, v0.6.1
atom-eco, v0.2.0
atom-handlebars, v0.4.0
browser-refresh, v0.8.3
coffee-lint, v0.7.3
html2haml, v0.6.0
language-cjsx, v0.2.0
language-eco, v0.3.0
language-ejs, v0.1.0
language-haml, v0.17.0
language-iced-coffee-script, v0.1.0
linter-rubocop, v0.2.2
linter-scss-lint, v0.0.14
policeman-rubocop, v0.0.3
script, v2.18.0
view-tail-large-files, v0.1.13
wakatime, v4.0.1
web-view, v0.3.0

# Dev
No dev packages
```

**Comment by mark-hahn** [comment]
Thanks mucho.

This is fixed in 0.1.14.  I should point out that this was due to incorrect settings.  A plugin was missing a regex.

**Comment by mark-hahn** [comment]
Wow, I just noticed this is from three weeks ago.  Sorry.
